### DISCUSSION TITLE
I need a walkthrough

### DISCUSSION THREAD

**Discussion by PolishFood** [Dpeta/pesterchum-alt-servers#42] [Q&A]
I'm not very tech savvy lmao, so sorry if this is a stupid question

I downloaded the latest release for Mac, and now I have my folder here all set up... what now? I don't know how to run it or what to click on

<img width="945" alt="Screen Shot 2021-07-07 at 11 15 34 AM" src="https://user-images.githubusercontent.com/87082095/124785830-33206380-df15-11eb-9071-f42dcc0c8892.png">

**Comment by Dpeta** [comment]
I think you should just be able to run "pesterchum" :0c

**Reply by PolishFood** [reply]
When I run "pesterchum," it just shows a bunch of script and doesn't do anything

**Reply by Dpeta** [reply]
Hmm, what version of macOS are you running? :?c

**Reply by milotheguylo** [reply]
I'm a mac user and I'm having the same problem. After disabling all blocks it's just a ton of script and it says something about a library not loading/an image not found. I'm not good with computers at all please help lol :(

EDIT: I downloaded python on my computer and now the code runs different? I have no clue if that's a good or a bad sign but at least I'm not getting vague information about missing images anymore.

**Reply by Dpeta** [reply]
Hmm, well, the manual route might work if the build doesn't, just:
1. Install Python 3 https://www.python.org/downloads/macos/
2. Open a terminal and run ``python3 -m pip install pygame PyQt5 feedparser python-magic ostools requests`` (to install dependencies)
3. Download the source from https://github.com/Dpeta/pesterchum-alt-servers/archive/refs/heads/py3_pyqt5.zip 
4. Extract py3_pyqt5.zip
5. Run start.sh. (Or, alternatively, open terminal, navigate into the folder with ``cd``, and then run ``python3 pesterchum.py``)
